examples/bri-3/src/bri/zkp/components/merkleTree/merkleTree.ts

```diff
@@ -1,5 +1,7 @@
 import { PreciseProofs } from 'ew-precise-proofs-js';
+import { getLogger } from '../../../../../LogConfig';
```

Comment:
rem these samples please, we ll also refactor this merkletree to use the nestjs modules approach

Reply:
Updated